website/content/wgs/cnaiwg/onboarding/index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: CNCF Cloud Native AI Working Onboarding
```

Comment:
Could we just title this 'Onboarding'

The left navigation on the CN AI WG page has the almost the same initial text for all links `CNCF Cloud Native AI Working..`

Reply:
Sure, I was just trying to be consistent but thought the same thing.